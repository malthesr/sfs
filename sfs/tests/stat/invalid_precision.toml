args = [
  "stat",
  "-s",
  "sum,sum,sum",
  "--precision",
  "2,1",
  "tests/stat/two_individuals.sfs"
]
status = "failed"
