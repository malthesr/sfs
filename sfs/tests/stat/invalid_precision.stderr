number of precision specifiers must equal one or the number of statistics (found 2 precision specifiers and 3 statistics)
